csss-site/src/events/frosh/templates/frosh/2023/frosh.html

```diff
@@ -55,6 +55,7 @@
 			<img class="logo" src="{% static 'frosh_static/2023/images/flag_h.png' %}" alt="CSSS RETRO FROSH 2023">
 		</div>
 		<section>
+			<iframe width="640" height="360" src="https://youtube.com/embed/QQ6u_fkrK0I"></iframe>
```

Comment:
Consider centering this element in the page. You can do this by adding `display: block; margin: 0 auto;` to the css

Reply:
You may also wish to increase the size of the iframe video to something a little larger, such as 720x405